master/internal/api_workspace.go

```diff
@@ -156,12 +156,13 @@ func (a *apiServer) GetWorkspaceProjects(
 	// Construct the ordering expression.
 	startTime := apiv1.GetWorkspaceProjectsRequest_SORT_BY_LAST_EXPERIMENT_START_TIME
 	sortColMap := map[apiv1.GetWorkspaceProjectsRequest_SortBy]string{
-		apiv1.GetWorkspaceProjectsRequest_SORT_BY_UNSPECIFIED:   "id",
-		apiv1.GetWorkspaceProjectsRequest_SORT_BY_CREATION_TIME: "created_at",
+		// `p` is an alias of `project` which is defined in master/static/srv/get_workspace_projects.sql
```

Comment:
SQL couldnt identity the field

<img width="1728" alt="Screenshot 2023-06-26 at 3 38 37 PM" src="https://github.com/determined-ai/determined/assets/109113616/c5d1ce9b-af22-4fdc-884a-99b4be8e2957">